app/controllers/spree/users_controller_decorator.rb

```diff
@@ -1,3 +1,21 @@
 Spree::UsersController.class_eval do
   layout 'darkswarm'
+
+  prepend_before_filter :load_object, :only => [:edit_address, :update_address, :show, :edit, :update]
```

Comment:
Good idea, if you run rubocop through the whole project there so many offenses. How did you check you code?